client/styles/importer.less

```diff
@@ -1,3 +1,5 @@
+@cache-version: `process.env.npm_package_version` ;
```

Comment:
True. I just think that in general, it would be surprising to get different results from running a command with npm as you would from manually running the command yourself.